src/command/FileCommands.cxx

```diff
@@ -233,3 +239,103 @@ handle_read_comments(Client &client, Request args, Response &r)
 
 	gcc_unreachable();
 }
+
+InputStreamPtr
+find_stream_art(const char *directory, Mutex &mutex, Cond &cond)
```

Comment:
There should be API documentation explaining what `directory` means - i.e. it's an UTF-8 URI, not a local filesystem path in filesystem encoding.